components/tidb_query_datatype/src/def/field_type.rs

```diff
@@ -195,9 +195,16 @@ pub trait FieldTypeAccessor {
     }
 
     /// Whether the flag contains `FieldTypeFlag::UNSIGNED`
+    #[inline]
     fn is_unsigned(&self) -> bool {
         self.flag().contains(FieldTypeFlag::UNSIGNED)
     }
+
+    /// Whether the flag contaits `FieldTypeFlag::PARSE_TO_JSON`
+    #[inline]
+    fn is_parse_to_json(&self) -> bool {
```

Comment:
Looks like this is a flag that rarely used. Prefer to not assign a function for it.

Reply:
Function not removed?